embedded-workshop-book/src/intro.md

```diff
@@ -4,3 +4,9 @@
 
 ### in development
 ❗️ To do
+
```

Comment:
could we link the setup instructions (i.e. https://github.com/ferrous-systems/embedded-trainings-2020/blob/master/README.md) either here or at the beginning of the Beginner/Advanced Course chapters?